src/Summary/SummaryView.js

```diff
@@ -60,29 +59,30 @@ class SummaryView extends React.Component {
             value={getLanguage}
           />
         </Col>
-        {PrintBoolToCheckbox('ui-organizations.summary.isVendor', dataVal.isVendor, 4)}
+        {PrintBoolToCheckbox('ui-organizations.summary.isVendor', initialValues.isVendor, 4)}
         <Col xs={12}>
           <KeyValue
             label={<FormattedMessage id="ui-organizations.summary.description" />}
-            value={get(dataVal, 'description', '')}
+            value={get(initialValues, 'description', '')}
           />
         </Col>
         <Col xs={12} className={css.rowHeader}>
           <div className={css.subHeadings}>
-            <b>
-              <FormattedMessage id="ui-organizations.summary.alternativeNames" />
-            </b>
+            <b><FormattedMessage id="ui-organizations.summary.alternativeNames" /></b>
```

Comment:
I think that's kind of weird to use classes and style tags in one place. Let's keep it for now, since it was here, but anyway...

Reply:
actually it's not weird when tags are used for SEO, styles don't provide anything to crawlers

Reply:
I'm not sure that `<b>` is important even for SEO. Maybe `H#` tags or semantics like `<header>`...